web/service/filetransfer.py

```diff
@@ -58,6 +57,7 @@ def handler(self, data):
             self._tap.put(msg)
 
     def worker_start(self):
+        from web import app
```

Comment:
I'm not a fan of this change.

I realize mutual imports are a bad pattern, but it's what the flask devs suggest for small or medium-sized apps, and it was manageable.

Having `import` in several functions is also pretty rough. Is there a particular reason to do it this way?

Reply:
I was running into issues with recursive imports causing app to fail to start; however, I'm pretty sure I figured that out and will put in a new commit to fix it.